addons/account/models/chart_template.py

```diff
@@ -87,21 +87,22 @@ class AccountChartTemplate(models.Model):
 
     name = fields.Char(required=True)
     parent_id = fields.Many2one('account.chart.template', string='Parent Chart Template')
-    code_digits = fields.Integer(string='# of Digits', required=True, default=6, help="No. of Digits to use for account code")
+    _default_code_digits = 6
+    code_digits = fields.Integer(string='# of Digits', required=True, default=_default_code_digits, help="No. of Digits to use for account code")
```

Comment:
I use it later (line 275), because it is was a hardcoded default value I preferred to let it declared here and because I hate to see used as hardcoded values used and not constants (it was the case in 15.0 version of this file in the *_prepare_transfer_account_for_direct_creation* and I didn't checked properly (yet) the file in master. 
It would break the code in several places if that default was ever changed. The idea was to keep the behavior but avoiding adding a possible breaking point if it changed.
Should I remove the default value (that would make the handling of all the impacted fields cleaner), let it like that (or let it set to 6 and use 6 on  line 275 but I really don't like that one)?